front/pages/api/v1/w/[wId]/assistant/conversations/[cId]/messages/index.ts

```diff
@@ -71,8 +71,21 @@ async function handler(
 
       const { content, context, mentions, blocking } = bodyValidation.right;
 
+      let userAuth: Authenticator = auth;
```

Comment:
I would keep using `auth` and just replace it in the condition below such that someone else editing this file won't miss using userAuth